app/jobs/scheduled/admin_problems.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Jobs
+  class AdminProblems < ::Jobs::Scheduled
+    every 30.minutes
+
+    def execute(args)
+      Notification.where(notification_type: Notification.types[:admin_problems]).destroy_all
+
+      if persistent_problems?
+        Group[:admins].users.each do |user|
+          Notification.create!(
+            notification_type: Notification.types[:admin_problems],
+            user_id: user.id,
+            data: "{}",
+          )
+        end
+      end
+    end
```

Comment:
I think this will delete and recreate the notification every 30 minutes until the problem is fixed? Is that how we want it to work?

Reply:
When I implemented this, I did not see a problem with that, but I believe it can be quite annoying. I have updated the PR.